Package.swift

```diff
@@ -11,7 +11,10 @@ let package = Package(
             dependencies:[]),
         Target(
             name:"ChatProviders",
-            dependencies:["Mapper", "BotsKit"]),
+            dependencies:["Mapper", "BotsKit", "ReplyService"]),
+        Target(
+            name:"ReplyService",
```

Comment:
I like idea of merging ChatProviders into BotsKit and also SwiftBot needs to be renamed as well.

As for Facebook + GraphQL also valid idea, one can be part of another